assets/src/amp-story-editor-blocks.js

```diff
@@ -331,7 +333,21 @@ store.subscribe( () => {
 	}
 } );
 
-registerPlugin( 'amp-story-featured-image-pre-publish', { render: withPrePublishNotice } );
+registerPlugin(
+	'amp-story-featured-image-pre-publish',
+	{
+		render: () => {
+			return (
+				<PrePublishPanel
+					validationCallback={ hasMinimumStoryPosterDimensions }
+					missingMediaMessage={ __( 'Selecting a featured image is required.', 'amp' ) }
+					invalidMediaMessage={ __( 'The featured image must have minimum dimensions of 696 by 928 pixels.', 'amp' ) /* @todo This is not accurate. The width is now 1200 pixels. */ }
```

Comment:
I think we should introduce a `getMinimumStoryPosterDimensions` function similar to `getMinimumFeaturedImageDimensions` and then use the returned dimensions in a string format to avoid duplicating the numbers across the codebase.